packages/cli/lib/errorHandlers/apiErrors.js

```diff
@@ -251,6 +254,13 @@ function logApiStatusCodeError(error, context) {
  * @param {ApiErrorContext}          context
  */
 function logApiErrorInstance(error, context) {
+  // Use the new local-dev-lib error handler
+  // NOTE: This will eventually replace the logic in logApiStatusCodeError
+  if (error.isAxiosError) {
+    const errorWithContext = getAxiosErrorWithContext(error, context);
+    logger.error(errorWithContext.message);
```

Comment:
I tried using `logErrorInstance` here, but it wants to prepend "An Error occurred." before every error message.